### PR TITLE
Perf-dash: add an option to override build count

### PR DESCRIPTION
**Pull request by oxddr**
This is useful for development, as it speeds up perf-dash startup by limiting number of builds it process. For simple correctness testing small values are fine.

Also: 

 * small refactoring of flag parsing
 * make target to run perf-dash locally

/assign @krzysied 
/area perf-dash

### CODE REVIEW

File: perfdash/Makefile [hunk 1]
```diff
@@ -8,6 +8,16 @@ REPO = gcr.io/k8s-testimages
 perfdash: perfdash.go parser.go config.go downloader.go google-gcs-downloader.go
 	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o perfdash
 
+run: perfdash
+	./perfdash \
+		--www \
+		--address=0.0.0.0:8080 \
+		--builds=20 \
+		--override-build-count \
+		--configPath=https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml
```
Comment (krzysied) [inline perfdash/Makefile]: IMHO `\` should be in this line, not the line below.

**Comment by krzysied** [comment]
/assign @mm4tt 
We need approval for known-flags.txt

File: perfdash/perfdash.go [hunk 2]
```diff
@@ -32,22 +32,27 @@ const (
...
+	www    = pflag.Bool("www", false, "If true, start a web-server to server performance data")
+	wwwDir = pflag.String("dir", "www", "If non-empty, add a file server for this directory at the root of the web server")
 
 	storageUrl = pflag.String("storageUrl", "https://k8s-gubernator.appspot.com/build", "Name of the data bucket")
 
 	globalConfig = make(map[string]string)
 )
 
+func initGoogleDownloaderOptions() {
+	pflag.BoolVar(&options.OverrideBuildCount, "override-build-count", false, "Whether to override number of build to process defined by \"perfDashBuildsCount\"")
```
Comment (mm4tt) [inline perfdash/perfdash.go]: Sorry, but for someone who has no knowledge of perf-dash code (me), this is really hard to understand. 
Could you define what this flag/feature does without referring to existing perf-dash intricacies (i.e. perfDashBuildsCount - I have no idea what it is / where it is defines)?

Comment (oxddr) [reply perfdash/perfdash.go]: Changed flags' name and  description to be hopefully easier to understand by someone not familiar with PerfDash.

**Comment by oxddr** [comment]
PTAL

**Pull request merged**